packages/electron-publish/src/s3/s3Publisher.ts

```diff
@@ -61,6 +61,10 @@ export class S3Publisher extends BaseS3Publisher {
     if (this.info.encryption != null) {
       args.push("--encryption", this.info.encryption)
     }
+
+    if (this.info.forcePathStyle != undefined) {
```

Comment:
Just to be consistent with the rest of the codebase, can you please make this `!= null`?